service/block/types.go

```diff
@@ -10,23 +11,33 @@ import (
 // it is still awaiting erasure coding.
 type Raw = core.Block
 
-// ExtendedBlock contains the erasure coded block data as well as its
+// Block represents the entirety of a Block in the Celestia network.
+// It contains the erasure coded block data as well as its
 // ExtendedHeader.
-// TODO @renaynay: ExtendedBlock still needs to be spec'd out.
-type ExtendedBlock struct {
-	// TODO @renaynay: include pointer to ExtendedHeader
-	data *ExtendedBlockData
+type Block struct {
+	header     *header.ExtendedHeader
+	data       *ExtendedBlockData
+	lastCommit *core.Commit
 }
 
 // ExtendedBlockData is an alias to rsmt2d's ExtendedDataSquare type.
-// TODO @renaynay: ExtendedBlockData still needs to be spec'd out.
 type ExtendedBlockData = rsmt2d.ExtendedDataSquare
```

Comment:
I am wondering if an alias is really needed here? I would suggest importing `rsmt2d` instead everywhere its needed instead of an alias

Reply:
I mean it's nice to call it "block data" rather than a square IMO.

Reply:
Ok